infra/resources/_modules/cosmos/io_com_cosmos.tf

```diff
@@ -0,0 +1,72 @@
+module "io_com_cosmos_account" {
+  source  = "pagopa/dx-azure-cosmos-account/azurerm"
+  version = "~> 0"
+
+  subnet_pep_id = var.subnet_pep_id
+
+  alerts = {
+    enabled         = true
+    action_group_id = var.action_group_id
+    thresholds = {
+      provisioned_throughput_exceeded = 1000
+    }
+  }
+
+  environment = var.environment
+
```

Comment:
I also suggest to add a secondary location as well
```hcl
secondary_geo_locations = [
  {
    location               = "spaincentral"
    failover_priority  = 1
    zone_redundant = false
  }
]
```

Reply:
Thanks [7a3d25d](https://github.com/pagopa/io-messages/pull/88/commits/7a3d25d9d3b13f54e65d2f912d74544021961eab)

Reply:
It's another variable, not included into `environment`

Reply:
[7a3d25d](https://github.com/pagopa/io-messages/pull/88/commits/7a3d25d9d3b13f54e65d2f912d74544021961eab)